### PR TITLE
A sample implementaion of  calling C++ SGX SDK QuoteVerify function using JNI

### PR DESCRIPTION
**Pull request by bronzeMe**
A sample implementaion of  calling C++ SGX SDK QuoteVerify function using JNI, and it works correctly

### CODE REVIEW

File: ppml/ppml-java-x86_64-linux/Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+include ../../Makefile.common
+
+CFLAGS = -I ${JAVA_HOME}/include/linux 
+SUFFIX = so
+
+all:
+ifeq ($(PLATFORM), LINUX)
+include ../Makefile.common
+endif
```
Comment (qiyuangong) [inline ppml/ppml-java-x86_64-linux/Makefile:9]: Add an empty line

File: ppml/src/main/cpp/quote_verification/App/quote_verification.h [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * Signature: ([BZ)I
+ */
+JNIEXPORT jint JNICALL Java_com_intel_analytics_ppml_attestation_QuoteVerifier_jni_1ecdsa_1quote_1verification
+  (JNIEnv *, jobject, jbyteArray);
+
+#ifdef __cplusplus
+  }
+#endif
+
+#endif /*QUOTE_VERIFICATION_H*/
```
Comment (qiyuangong) [inline ppml/src/main/cpp/quote_verification/App/quote_verification.h:24]: Add an empty line

File: ppml/src/main/cpp/quote_verification/App/quote_verification.cpp [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+#define SAMPLE_ISV_ENCLAVE  "enclave.signed.dll"
+#define DEFAULT_QUOTE   "..\\..\\..\\QuoteGenerationSample\\x64\\Debug\\quote.dat"
+
+#define strncpy	strncpy_s
+#endif
+
+
+using namespace std;
+
+void test_so()
```
Comment (qiyuangong) [inline ppml/src/main/cpp/quote_verification/App/quote_verification.cpp:31]: Remove this test. Or put it in test dir

File: ppml/src/main/cpp/quote_verification/App/quote_verification.cpp [hunk 4]
```diff
@@ -0,0 +1,244 @@
+#include <stdio.h>
```
Comment (qiyuangong) [inline ppml/src/main/cpp/quote_verification/App/quote_verification.cpp:1]: App is not necessary. 
We can use this path https://github.com/intel-analytics/BigDL-core/tree/master/ppml/src/main/java/com/intel/analytics/bigdl/ppml/attestation

File: ppml/src/main/java/com/intel/analytics/bigdl/ppml/attestation/QuoteVerifyJNICmd.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
+/*
```
Comment (qiyuangong) [inline ppml/src/main/java/com/intel/analytics/bigdl/ppml/attestation/QuoteVerifyJNICmd.java:1]: We can move this QuoteVerifyCmd to BigDL.

File: ppml/src/main/cpp/quote_verification/App/quote_verification.cpp [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+        ret = -1;
+        break;
+    }
+
+    return ret;
+}
+
+void usage()
+{
+    printf("\nUsage:\n");
```
Comment (qiyuangong) [inline ppml/src/main/cpp/quote_verification/App/quote_verification.cpp:159]: Usage is not necessary anymore

**Comment by qiyuangong** [comment]
Hi @bronzeMe 
Thank you for contribution!

The whole PR looks good. Need to clean up unnecessary methods, e.g., usage, test.

Also, we need to add reference for SimpleCode at README.md (https://github.com/intel-analytics/BigDL-core/tree/master/ppml)

**Comment by bronzeMe** [comment]
> Hi @bronzeMe Thank you for contribution!
> 
> The whole PR looks good. We can make it better if we can:
> 
> 1. Clean up unnecessary methods, e.g., usage, test.
> 2. Remove App in package path.
> 
> Also, we need to add reference for SimpleCode at README.md (https://github.com/intel-analytics/BigDL-core/tree/master/ppml)

@qiyuangong ,Please review the updated PR, thanks a lot.

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**